autopush/endpoint.py

```diff
@@ -435,7 +440,7 @@ def _uaid_lookup_results(self, result):
 
         # Only simplepush uses version/data out of body/query, GCM/APNS will
         # use data out of the request body 'WebPush' style.
-        if router_key == "simplepush":
+        if not self._is_webpush():
```

Comment:
Function calls are still expensive on cpython, so I'd prefer to avoid easily avoidable ones like this in a hot path until we're on pypy.